### ISSUE TITLE
Sort email and recipient group lists by modified date

### ISSUE THREAD

**Issue by cjg89** [UCF/PostMaster#74]
Update lists of emails and recipient groups in the postmaster admin to list objects by their last-modified date, with most recently-updated objects appearing first, so that we can more quickly access emails and groups we're actively working on instead of having to click the "modified date" filters twice.

**Comment by jmbarne3** [comment]
Just to clarify, we want the lists to be sorted by the last-modified date by default?

**Comment by cjg89** [comment]
Yes, last-modified by default would be ideal.  Currently it looks like both the email and recipient group list views are sorted by name.
